### DISCUSSION TITLE
Date Enabled as a mod conflict tool?

### DISCUSSION THREAD

**Discussion by ZLRider** [X2CommunityCore/xcom2-launcher#381] [Ideas]
Turning on a date-enabled tab like the date-installed tab would be a big help when trying to identify which new mod is conflicting with another when a player is running a lot of mods. I don't know if that is possible as I'm not a programmer by any stretch.

**Comment by RevZero** [comment]
I don't think implementing something like that is worth the trouble. Its a good practice not to add/enable too many mods at once. And even if you had an indicator like that, it still wouldn't help if you just enabled 10 mods. You would still need to figure out which one of those causes the issue.
